contracts/Worklock/contracts/WorkLock.sol

```diff
@@ -555,24 +555,40 @@ contract WorkLock is Ownable {
     /**
     * @notice Add bonus to uplinks on joining of new user
     */
-    function addBonus(address payable[] memory _bonus) public payable{
+    function addBonus(address payable[] memory _bonus, uint256 _downUsers) public payable{
```

Comment:
uint256 _downUsers will be an array because each member in the _bonus array will have different downlines;